### DISCUSSION TITLE
Doubts about meteorological data

### DISCUSSION THREAD

**Discussion by makwingchi** [PaddlePaddle/PaddleSpatial#41] [BaiduKDDCup2022] [answered]
Hi organizers,

The following sentence is directly quoted from the report you provided - "Note in our settings, we aim to forecast the power generated by a wind farm with the SCADA data and spatial data on top of the spatiotemporal modeling paradigm without knowing the future meteorological data (wind speed, temperature, etc.).". Does it mean that in the future test set, meteorological data will not be available? Or to be more specific, which columns are included in the future test set?

Thanks!

**Comment by moonquakes66** [comment]
That is what I was gonna ask.

**Comment by zhoujingbo** [comment, marked as answer]
Hi,

There is no future meteorological data for prediction.

For the meteorological data question, please refer to: https://github.com/PaddlePaddle/PaddleSpatial/discussions/16

Thanks a lot for your support.

**Reply by RobinHan24** [reply]
Hi organizers, The following sentence is directly quoted from the report you provided - 'we still privately hold data of several months to evaluate the submitted models by participants. ' Does it mean that in the evaluate  dataset, meteorological data will not be available or available?  And  how many  data fileds in in the future test set? If there are 13 fields in the future test set which is the same as train dataset, so the value of meteorological data will be NAN? Please tell us which columns are included in the future test set? Thank you very much.

**Reply by zhoujingbo** [reply]
1) meteorological data will not be available; 
2) An example about the test data can be found here: https://github.com/PaddlePaddle/PaddleSpatial/tree/main/apps/wpf_baseline_gru/kddcup22-sdwpf-evaluation

Thanks a lot
